python/torch_mlir_e2e_test/configs/tcp_backend.py

```diff
@@ -0,0 +1,50 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+from typing import Any
+
+import torch
+import torch_mlir
+
+from torch_mlir_e2e_test.tcp_backends.abc import TcpBackend
+from torch_mlir_e2e_test.framework import TestConfig, Trace, TraceItem
+from torch_mlir_e2e_test.utils import convert_annotations_to_placeholders
+from .utils import (
+    recursively_convert_to_numpy,
+    recursively_convert_from_numpy,
+)
+
+
+class TcpBackendTestConfig(TestConfig):
+    """Base class for TestConfig's that are implemented with linalg-on-tensors.
+
+    This class handles all the common lowering that torch-mlir does before
+    reaching the linalg-on-tensors abstraction level.
```

Comment:
i think linalg-on-tensors should be tcp

Reply:
I'm not sure what you are referring to here. This is very similar to MhloBackendTestConfig and TosaBackendTestConfig IIUC.